deploy/role.yaml

```diff
@@ -15,6 +15,27 @@ rules:
   - secrets
   verbs:
   - "*"
+- apiGroups:
+  - ""
+  resources:
+  - podtemplates
+  - namespaces
+  - componentstatuses
+  - nodes
```

Comment:
As per my understanding, `nodes/namespaces` can be put in `cluster_role.yaml` only, we don't need it to be available in the role

Reply:
Also wondering why would we need nodes even for reading/listing in cluster role... But maybe there are some indirect dependencies?

Reply:
We should need only list permission for nodes - it's needed by KubeFed controller that checks the health of the clusters.
This is already part of the `cluster_role.yaml` file, so no need for additional changes